python/src/LinearModelValidation_doc.i.in

```diff
@@ -0,0 +1,152 @@
+%feature("docstring") OT::LinearModelValidation
+"Validate a linear regression metamodel.
+
+Available constructors:
+    LinearModelValidation(*linearModelResult, cvMethod*)
+    
+    LinearModelValidation(*linearModelResult, cvMethod, kParameter*)
```

Comment:
here they can be dropped too

Reply:
Ok. Fixed at 9c0f7ba6dacbaeec69233d91fa049c36fd0bf3fe